src/main/java/com/google/sps/data/prioritizeCoursesCriteria.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+public class prioritizeCoursesCriteria implements Preference {
+
+  public void prioritizeCoursesCriteria(List<Course> courseList) {
+    this.courseList = courseList;
+  }
+
+  List<Course> courseList;
```

Comment:
nit: you can choose if you want to change this (according to the google style guide the way you're doing it is 100% valid), but Heidi and I both declare our fields before the constructor. It might be worth switching for style consistency throughout the project.

Reply:
Ah ok thats fair. I'll do that to all the criteria classes.